cli/config.go

```diff
@@ -44,10 +44,14 @@ type channel struct {
 	Topic  string `toml:"topic"`
 }
 
+type user_token struct {
+	user_token string `toml:"user_token"`
+}
 type config struct {
-	Remotes remotes `toml:"remotes"`
-	Filter  filter  `toml:"filter"`
-	Channel channel `toml:"channel"`
+	Remotes    remotes    `toml:"remotes"`
+	Filter     filter     `toml:"filter"`
+	Channel    channel    `toml:"channel"`
+	User_token user_token `toml:"user_token"`
```

Comment:
`UserToken`. Do we need a particular structure or can we simply use string?

Reply:
Please follow Go idiomatic naming - CammelCase, not snake_case

Reply:
Okay

Reply:
I have addressed the comments